### ISSUE TITLE
Missing Structures and Type Parameters

### ISSUE THREAD

**Issue by jonasnordlund** [leddt/visualstudio-colors-solarized#44]
I noticed my Visual Studio 2017 was listing _User Types - Structures_ and _User Types - Type Parameters_ without custom colors, so I checked with the template and it didn't have those in the list next to the other user types.

According to the ReSharper settings also (needlessly in 2017?) having those, Structures should use foreground $Yellow and Type Parameters should use $Magenta. Or in .settings XML:

```XML
<Item Name="type parameter name" Foreground="0x008236D3" Background="0x02000000" BoldFont="No" />
<Item Name="struct name" Foreground="0x000089B5" Background="0x02000000" BoldFont="No" />
```

I also note how my Visual Studio 2017 writes out its .settings using different item names. I'm not sure what's up with that. Maybe Visual Studio 2017 is backwards compatible with different 2015 names or something? For example, upon exporting, I don't see _"User Types - Classes"_ as in the template used here and what I have successfully imported, but I do get _"class name"_.

**Comment by jonasnordlund** [comment]
I was going to file another report for missing XML Doc Comment Attributes as they are in glaring white to me due to falling back to default values, but upon browsing even more properties I see even more that is missing like various XAML markup (about half is supported), VBScript support, and more. All of these are falling back to whatever was the default in the theme you were importing these settings into as they will remain unchanged. Seems like a greater overhaul would be in order.

**Comment by leddt** [comment]
Yeah, I really haven't been maintaining this properly since around 2012. I now just use the default dark theme in VS, so I'm relying on pull requests from users of the theme to keep it up to date.

**Comment by jonasnordlund** [comment]
OK, thanks for acknowledging the state of things, I'll see if I can contribute with something. :-)

**Comment by mario-d-s** [comment]
@jonasnordlund Have you looked into this in the meantime?

I'm currently investigating the contents of `.vssettings` to try and figure out how to maintain this template for future versions (and whether anything is missing for the current version). See also #45.

Currently diffing it with mine generated from Visual Studio Professional 2017 version 15.6.5. What version are you on?
